### ISSUE TITLE
Non-standard SSH port requires addition of quotes in rsnapshot script

### ISSUE THREAD

**Issue by hmep** [rsnapshot/rsnapshot#212]
For some time, rsnapshot has not been working properly on a server of mine (Ubuntu 16.04.4 LTS), when using the following config (which uses a non-standard SSH port):

```
backup          user@server.tdl:/path/to/data        data/   ssh_args=-p23
```

It seems that the following edit to the rsnapshot script does the trick:

```
# diff /usr/bin/rsnapshot /usr/bin/rsnapshot.bak 
3737c3737
<                       push(@rsync_long_args_stack, "--rsh='$config_vars{'cmd_ssh'} $ssh_args'");
---
>                       push(@rsync_long_args_stack, "--rsh=$config_vars{'cmd_ssh'} $ssh_args");
```

**Comment by mathintro** [comment]
Yep. It indeed laso does the trick on the latest Arch Linux.

**Comment by sam-at-github** [comment]
What version, config, and verbose error output. I cannot reproduce this error in 1.4.2.

A long time ago there (#15, #25) there used to be quotes. Now there is no quotes, and I believe the fix in #15 is correct.
